src/module/item/physical/sheet.ts

```diff
@@ -26,22 +26,30 @@ export class PhysicalItemSheetPF2e<TItem extends PhysicalItemPF2e = PhysicalItem
             { rollData, async: true }
         );
 
+        const stackGroup = sheetData.data?.stackGroup;
+
         return {
             ...sheetData,
             itemType: game.i18n.localize("PF2E.ItemTitle"),
             basePriceString: new CoinsPF2e(this.item._source.system.price.value).toString(),
             priceString: this.item.price.value.toString(),
             actionTypes: CONFIG.PF2E.actionTypes,
             actionsNumber: CONFIG.PF2E.actionsNumber,
+            bulkTypes: CONFIG.PF2E.bulkTypes,
             frequencies: CONFIG.PF2E.frequencies,
             sizes: CONFIG.PF2E.actorSizes,
+            stackGroups: CONFIG.PF2E.stackGroups,
+            usage: CONFIG.PF2E.usageTraits,
             isPhysical: true,
             activations: this.item.activations.map((action) => ({
                 action,
                 id: action.id,
                 base: `system.activations.${action.id}`,
                 traits: createSheetTags(actionTraits, action.traits ?? { value: [] }),
             })),
+
+            // Do not let user set bulk if in a stack group because the group determines bulk
+            bulkDisabled: stackGroup !== undefined && stackGroup !== null && stackGroup.trim() !== "",
```

Comment:
```suggestion
            bulkDisabled: !stackGroup?.trim(),
```

Reply:
I really should have tried to understand what it was doing rather than trusting it 🙃

Reply:
Should be !!, I'll force push it in.